### PR TITLE
Add two new options to .../smores line of story

### PR DESCRIPTION
**Pull request by JacqL**

### CODE REVIEW

**Comment by JacqL** [comment]
Please review this, @casey-collab

File: english/invite-friends/smores/after_smores/indigestion.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Seating back after those satisfying smores, you suddenly get very tired.
```
Comment (casey-collab) [inline english/invite-friends/smores/after_smores/indigestion.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/invite-friends/smores/after_smores/shrinking_room.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+OK, so those smores were darn good.
```
Comment (casey-collab) [inline english/invite-friends/smores/after_smores/shrinking_room.md]: This file has also been added to a directory that already contained a page.

File: english/invite-friends/smores/with_friends.md [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
 You place the hot marshmallow and a piece of chocolate on the graham
-cracker and enjoy your new snack with your friends.
\ No newline at end of file
+cracker and enjoy your new snack with your friends.
+
+Now what?
+
+Do you want to:
+
+- [Sit back and relax](smores/after_smores/indigestion.md)
```
Comment (casey-collab) [inline english/invite-friends/smores/with_friends.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/invite-friends/smores/with_friends.md [hunk 4]
```diff
@@ -1,4 +1,13 @@
...
+
+- [Figure out what these marshmellow walls are all about?](smores/after_smores/shrinking_room.md)
```
Comment (casey-collab) [inline english/invite-friends/smores/with_friends.md]: This link is also invalid.

File: english/invite-friends/smores/after_smores/indigestion.md (hunk 1 above)
Comment (casey-collab) [inline english/invite-friends/smores/after_smores/indigestion.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

File: english/invite-friends/smores/after_smores/shrinking_room.md (hunk 2 above)
Comment (casey-collab) [inline english/invite-friends/smores/after_smores/shrinking_room.md]: This new file also has no link to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

**Comment by casey-collab** [comment]
Looks great! :night_with_stars:

**Pull request merged**